src/org/zaproxy/zap/extension/params/ExtensionParams.java

```diff
@@ -349,6 +352,17 @@ public boolean onHttpResponseReceive(HttpMessage msg) {
 			persist(sps.addParam(site, iter.next(), msg));
 		}
 
+		// Header "Parameters"
+		List<HttpHeaderField> headersList = msg.getResponseHeader().getHeaders();
+		String[] setCookieHeaders = {HttpHeader.SET_COOKIE, HttpHeader.SET_COOKIE2};

```

Comment:
Could be converted to List here, also, we should lower case these headers and the one being checked (to ensure that we do a case insensitive check).